lib/kasket/write_mixin.rb

```diff
@@ -58,6 +58,19 @@ def kasket_keys
         keys
       end
 
+      def kasket_after_commit
+        keys = kasket_keys
+
+        if persisted? && Kasket::CONFIGURATION[:write_through]
```

Comment:
I was trying to capture whether the record was being created/updated or destroyed, to avoid having separate `after_commit on => :destroy`, `after_commit on: [:create, :update]` hooks

In my testing, `persisted?` was `true` when creating/updating/touching and `false` when destroying.